tasks/libvirt_default_net_ipv6_tasks.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Parse libvirt default network configuration
+  virt_net:
+    command: get_xml
+    name: default
+  register: default_net_xml
+- debug: var=default_net_xml.stdout
+- name: Remove IPv4 configuration
+  xml:
+    xmlstring: "{{ default_net_xml.get_xml }}"
+    xpath: /network/ip
+    state: absent
+  register: editednet_noipv4
+- name: Configure it as an isolated network
+  xml:
+    xmlstring: "{{ editednet_noipv4.xmlstring }}"
+    xpath: /network/forward
+    state: absent
+  register: editednet_isolated
+- name: Edit libvirt default network configuration, 1
+  xml:
+    xmlstring: "{{ editednet_isolated.xmlstring }}"
+    xpath: /network/ip[@family='ipv6']
+    attribute: address
+    value: "{{ he_ipv6_subnet_prefix + '::1' }}"
+  register: editednet1
+- name: Edit libvirt default network configuration, 2
+  xml:
+    xmlstring: "{{ editednet1.xmlstring }}"
+    xpath: /network/ip[@family='ipv6']
+    attribute: prefix
+    value: "64"
+  register: editednet2
+- debug: var=editednet2
+- name: Edit libvirt default network configuration, 3
+  xml:
+    xmlstring: "{{ editednet2.xmlstring }}"
+    xpath: /network/ip[@family='ipv6']/dhcp/range
+    attribute: start
+    value: "{{ he_ipv6_subnet_prefix + '::10' }}"
+  register: editednet3
+- debug: var=editednet3
+- name: Edit libvirt default network configuration, 4
+  xml:
+    xmlstring: "{{ editednet3.xmlstring }}"
+    xpath: /network/ip[@family='ipv6']/dhcp/range
+    attribute: end
+    value: "{{ he_ipv6_subnet_prefix + '::ff' }}"
+  register: editednet4
+- debug: var=editednet4
+- name: Update libvirt default network configuration, 1
```

Comment:
same - we can name the tasks with something like: destroy, undefine and define

Reply:
done